pydatastructs/strings/tests/test_trie.py

```diff
@@ -23,7 +33,7 @@ def test_Trie():
         trie.delete(string)
         for present in strings:
             if present == string:
-                assert not trie.is_present(present)
+                assert not trie.is_inserted(present)
             else:
-                assert trie.is_present(present)
+                assert trie.is_inserted(present)
```

Comment:
Why call to is_present is removed? I think we should have both.

Reply:
I think it's because the previous tests were actually meant to test `is_inserted`, and the bug is caught when we use `is_present`.
For example, if we have already deleted `"A"` from the trie, then still `trie.is_present("A")` will be `true` which is logically correct since `"Amfn"` is present. But the previous tests had this bug so I think this looks fine.